README.md

```diff
@@ -17,7 +17,7 @@ You may read [here](https://github.com/microverseinc/curriculum-javascript/blob/
 
 ![screenshot](JavaScript-Capstone-LIveDemo.png)
 
-## [LiveDemo](https://fatima-hub333.github.io/Capstone-Movies-Api/dist/)
+## [LiveDemo](https://fatima-hub333.github.io/Capstone-Movies-Api)
```

Comment:
- [ ] [OPTIONAL] You can add /dist at the end of the link for your link to work directly because now it is taking me to the README file